ExeterTeam25/group-software-project
# Alerts user to change their username

To test:
- remove your sso sign in from the db if one exists
- sign in with sso
- you should be alerted to change your username

Details:
- alerts user to change their username when signing in with sso for the first time
